netlogo-gui/src/main/properties/EditPanel.scala

```diff
@@ -62,6 +62,7 @@ class EditPanel(val target: Editable, val compiler: CompilerServices, colorizer:
       c.anchor = GridBagConstraints.WEST
       c.insets = new Insets(3, 3, 3, 3)
       c.gridwidth = property.gridWidth
+      if (editor.isInstanceOf[BooleanEditor]) c.fill = GridBagConstraints.HORIZONTAL
```

Comment:
This did fix the squishing issue, but ideally the editor panel shouldn't have to know anything about how the specific property editors are setup.  Any way to get the `HORIZONTAL` constraint setting moved into the `BooleanEditor` itself?